Update functions.md (#3421)
Codecademy/docs: content/cpp/concepts/functions/functions.md

  std::cout << a;
}
```

## Overloading

The ability to create multiple functions with the same name but different parameters is called function [overloading](https://www.codecademy.com/resources/docs/cpp/overloading) in C++. It provides a way to reuse the same function name in different contexts by accepting different data types and numbers of arguments.

```cpp
#include <iostream>
using namespace std;

void add(int a,int b){

>   cout << "Output,when add function is called with integer parameters: " << a+b << "\n";

Reviewer: ```suggestion
  cout << "Output, when add function is called with integer parameters: " << a+b << "\n";
```

}

void add(string a, string b){

>   cout << "Output,when add function is called with string parameters: " << a+b << "\n";

Reviewer: ```suggestion
  cout << "Output, when add function is called with string parameters: " << a+b << "\n";
```

}

int main() {
  int a=10,b=10;
  string s = "Hello",t="World!";
  add(a,b);
  add(s,t);
}

> 

Reviewer: remove extra empty line

```

> This will Output:

Reviewer: ```suggestion
This will output:
```

> ```

Reviewer: ```suggestion
```shell
```

  std::cout << a;
}
```

## Overloading

> The ability to create <strong>multiple functions</strong> with the <strong>same name</strong> but <strong>different parameters</strong> is called function overloading in C++. It provides a way to <strong>reuse the same function</strong> name in different contexts by accepting different data types and numbers of arguments.

Reviewer 2: ```suggestion
The ability to create multiple functions with the same name but different parameters is called function [overloading](https://www.codecademy.com/resources/docs/cpp/overloading) in C++. It provides a way to reuse the same function name in different contexts by accepting different data types and numbers of arguments.
```
This is a Markdown document, please don't use HTML in it.

```cpp
#include <iostream>
using namespace std;

void add(int a,int b){

>   cout<<"Output,when add function is called with integer parameters: "<<a+b <<"\n";

Reviewer 2: ```suggestion
  cout << "Output,when add function is called with integer parameters: " << a+b << "\n";
```

}

void add(string a, string b){

>   cout <<"Output,when add function is called with string parameters: " <<a+b <<"\n";

Reviewer 2: ```suggestion
  cout << "Output,when add function is called with string parameters: " << a+b << "\n";
```

}

int main() {
  int a=10,b=10;
  string s = "Hello",t="World!";
  add(a,b);
  add(s,t);
}

```

This will Output:

```
When add function is called with integer parameters: 20
When add function is called with string parameters: HelloWorld!
```

> 
> [Read More about Overloading](https://www.codecademy.com/resources/docs/cpp/overloading)

Reviewer 2: ```suggestion
```
Remove this last line. I moved the link to the first paragraph.
